spec/API_specification/array_object.rst

```diff
@@ -246,7 +246,7 @@ Bitwise Operators
 
 -------------------------------------------------
 
-.. currentmodule:: signatures.array_object
+.. currentmodule:: array_api.array_object
```

Comment:
If you change this to `currentmodule:: array_api`, does that make it so you can access things with a flat namespace. It would mean the categories aren't used in the docs at all (but they are still useful to have in the package because we use them in the test suite).

Reply:
> so you can access things with a flat namespace? It would mean the categories aren't used in the docs at all

That's precisely the desired effect that we'd like to achieve here!

Reply:
This has been resolved.